geode-tcp-server/src/test/java/org/apache/geode/distributed/internal/tcpserver/HostAndPortTest.java

```diff
@@ -14,19 +14,30 @@
  */
 package org.apache.geode.distributed.internal.tcpserver;
 
+import static org.apache.geode.internal.serialization.DataSerializableFixedID.HOST_AND_PORT;
 import static org.assertj.core.api.Assertions.assertThat;
 
+import java.io.IOException;
 import java.net.InetSocketAddress;
 
 import org.junit.Test;
+import org.junit.experimental.categories.Category;
 
+import org.apache.geode.internal.serialization.BufferDataOutputStream;
+import org.apache.geode.internal.serialization.ByteArrayDataInput;
+import org.apache.geode.internal.serialization.DSFIDSerializer;
+import org.apache.geode.internal.serialization.DSFIDSerializerFactory;
+import org.apache.geode.internal.serialization.Version;
+import org.apache.geode.test.junit.categories.MembershipTest;
+
+@Category({MembershipTest.class})
 public class HostAndPortTest {
 
   /**
    * Test that getSocketInentAddress returns resolved InetSocketAddress
    */
   @Test
-  public void Test_getSocketInentAddress_returns_resolved_SocketAddress() {
+  public void getSocketInentAddress_returns_resolved_SocketAddress() {
```

Comment:
there seems to be a misspelling here: "Inent"

Reply:
fixed